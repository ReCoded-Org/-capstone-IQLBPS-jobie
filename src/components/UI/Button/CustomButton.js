const CustomButton = ({ btnName }) => {
  return (
    <button
      type="button"
      className="w-25 h-7 bg-white text-accent border-2 
      border-accent rounded-3xl px-5 small:w-15 small:h-5 
      small:rounded-xs small:px-2 hover:border-primary hover:text-primary"
    >
      {btnName}
    </button>
  );
};
export default CustomButton;
