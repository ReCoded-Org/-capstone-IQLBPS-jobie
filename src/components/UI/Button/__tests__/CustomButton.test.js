import React from "react";
import renderer from "react-test-renderer";
import CustomButton from "../CustomButton";
it("renders Custom button correctly", () => {
  const tree = renderer.create(<CustomButton btnName="Any" />).toJSON();
  expect(tree).toMatchSnapshot();
});
