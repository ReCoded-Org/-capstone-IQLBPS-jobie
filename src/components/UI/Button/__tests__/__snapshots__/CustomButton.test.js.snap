// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Custom button correctly 1`] = `
<button
  className="w-25 h-7 bg-white text-accent border-2 
 border-accent rounded-3xl px-5 small:w-15 small:h-5 
 small:rounded-xs small:px-2 hover:border-primary hover:text-primary"
  type="button"
>
  Any
</button>
`;
